frontend/src/i18n/messages/en/index.ts

```diff
@@ -674,6 +674,39 @@ export const messages = {
               "Applying feedback from the community, we iterated on improvements that made it easier to adjust search filter criteria, share search results, and save relevant results and opportunities.",
           },
         ],
+        contentTitle2: "Late 2024",
```

Comment:
maybe we can use a more descriptive name here - "archived roadmap title" and "archived roadmap items"?